doc/source/whatsnew/v2.3.0.rst

```diff
@@ -36,7 +36,8 @@ Other enhancements
   when using ``np.array()`` or ``np.asarray()`` on pandas objects) has been
   updated to work correctly with NumPy >= 2 (:issue:`57739`)
 - The :meth:`~Series.sum` reduction is now implemented for ``StringDtype`` columns (:issue:`59853`)
--
+- Median percentile is only included in :meth:`~Series.describe` when a blank
+  list is passed (:issue:`60550`).
```

Comment:
Can you move this to `v3.0.0.rst`. We should not do this breaking change in 2.3.